en-US/Design.xml

```diff
@@ -161,7 +183,8 @@
 					<term>Source options <replaceable>[username@]hostname:/repository_filename</replaceable>)</term>
```

Comment:
I'd change the examples to use arguments instead of options to follow the libc quote above.